client/src/containers/User.jsx

```diff
@@ -100,7 +100,24 @@ class User extends Component {
     return (
       <div>
         <h2>{user}</h2>
-
+        {userPage.goal &&
+          isAuthenticated && (
+            <Link
+              to={{
+                pathname: '/progress/edit',
+                state: {
+                  currentCourse: `${userPage.currentCourse}`,
```

Comment:
why a template literal if there's nothing else other than the string (are all strings?)

Reply:
will it work without one?

Reply:
if those are all strings, yes.

Reply:
I had no idea, thanks!